bin/hdfcoinc/pycbc_make_coinc_search_workflow

```diff
@@ -155,9 +179,10 @@ ind_insps = insps = wf.setup_matchedfltr_workflow(workflow, science_segs,
 insps = wf.merge_single_detector_hdf_files(workflow, hdfbank[0], 
                                            insps, output_dir, tags=[tag])
 
-anal_log_files = wf.setup_analysislogging(workflow,
-                                          science_seg_file, insps, args,
-                                          "logging", program_name=__program__) 
```

Comment:
I'd suggest removing the module if it is no longer used. I don't suspect that anyone is using it without our knowledge.